src/ripple/core/Stoppable.h

```diff
@@ -362,7 +363,7 @@ RootStoppable::alertable_sleep_for(
     std::unique_lock<std::mutex> lock(m_);
     if (m_calledStop)
         return true;
-    return c_.wait_for(lock, d, [this]{return m_calledStop;});
+    return c_.wait_for(lock, d, [this]{return m_calledStop.load();});
```

Comment:
Yes.  It makes the return type of the lambda `bool` instead of `atomic<bool>`.  The latter won't compile because `atomic<bool>` is not copyable.